dcos: add 2.1.3 release notes

jkoelker opened the pull request:
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-73456

## Description of changes being made

Add release noted of 2.1.3


## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

msdolbey commented on pages/mesosphere/dcos/2.1/release-notes/2.1.3/index.md:
```diff
@@ -0,0 +1,44 @@
+- Etcd is now disabled when calico is disabled via `calico_enable` (D2IQ-73299).
+- Fixed dcos-net startup script to configure network ignore file for on-prem (D2IQ-73113).
+- Calico CNI will not be configured when `calico_enabled` is `false` (D2IQ-73141).
+- Package registry fixed to work with intermediate certificates. (D2IQ-72615).
```
I guess, because there's a period at the end of the parenthesis at the end of the line here, you don't need a period before the parenthesis?

```suggestion
- Package registry fixed to work with intermediate certificates (D2IQ-72615).
```

jkoelker replied:
Thanks!, yea I added the one at the end, after copying from the changelog, didn't see the other :doh:. Committed and squashed.

msdolbey reviewed (approved):
👍 
![Release it!](https://media1.tenor.com/images/297801e1ae1e4316e89227b7e64365f3/tenor.gif)

megatmar reviewed (changes requested):
i like that you have included the jira D2IQ- issue numbers, but there are a few more issues that need calling out (please see this dashboard:  https://jira.d2iq.com/secure/Dashboard.jspa?selectPageId=32524)

i believe the following issues also should appear in the release notes, if you please:

COPS-6561 is resolved by D2IQ-72615, and should be listed alongside it within the parentheses.
COPS-6519 doesn't have a corresponding D2IQ- issue number, but should also be included
(we always try to include COPS- issues in release notes)
finally, MARATHON-8771 may need to be included, as its fixVersion is 2.1.3, but this is a call for @vinodkone 

thank you!

cmastr commented:
> i like that you have included the jira D2IQ- issue numbers, but there are a few more issues that need calling out (please see this dashboard: https://jira.d2iq.com/secure/Dashboard.jspa?selectPageId=32524)
> 
> i believe the following issues also should appear in the release notes, if you please:
> 
> COPS-6561 is resolved by D2IQ-72615, and should be listed alongside it within the parentheses.
> COPS-6519 doesn't have a corresponding D2IQ- issue number, but should also be included
> (we always try to include COPS- issues in release notes)
> finally, MARATHON-8771 may need to be included, as its fixVersion is 2.1.3, but this is a call for @vinodkone
> 
> thank you!

Agree with this. Support needs COPS tickets listed specifically, and previously, @vinodkone also wanted bug fixes listed as well as COPS fixes.

cmastr commented on pages/mesosphere/dcos/2.1/release-notes/2.1.3/index.md:
```diff
@@ -0,0 +1,44 @@
+### DC/OS Fixed and Improved Issues
+DC/OS 2.1.3 fixes the following issues:
+
+- Etcd is now disabled when calico is disabled via `calico_enable` (D2IQ-73299).
```
Do any of these have matching COPS numbers?

jkoelker replied:
The etcd one there isn't a cops for, that's one i just noticed while chatting with a customer, the CNI one was also brought to our attention by a customer, but there wasn't a cops created since it was immediately apparent what the bug was.

jkoelker commented:
Thanks for the reviews! I added the relevant COPS #'s where they existed, as well as the Marathon one (since it doens't appear in the marathon changelog). Squashed and rebased it atop staging. \o/

jkoelker commented on pages/mesosphere/dcos/2.1/release-notes/2.1.3/index.md:
Also no cops for this one, it was a dependabot bump, that we had to manually fix, only included it here since its fixing an RSA CVE.
